pynamodb/attributes.py

```diff
@@ -77,8 +77,18 @@ def _initialize_attributes(cls):
         """
         cls._attributes = {}
         cls._dynamo_to_python_attrs = {}
-        for item_name, instance in six.iteritems(cls.__dict__):
-            if issubclass(type(instance), (Attribute,)):
+
+        for item_name in dir(cls):
```

Comment:
@lita `inspect` could have performance implications (even though its API is nicer). Not saying this will happen here but something to watch out for. See http://go.lyft.com/optimizing-python